### ISSUE TITLE
Minimal POST route results in 500 Internal Server Error

### ISSUE THREAD

**Issue by andreasbhansen** [architect/sandbox#106]
**Describe the issue**

I've made an app with a total of three routes: one `GET` and two `POST`. The `GET /` route works perfectly fine, however both the `POST /` and `POST /login` will return 500 Internal Server Error. 

My `.arc` file:
```
@app
arc-repro

@http
get /
post /
post /login
```

Both the `POST /` and `POST /login` has the same minimal content: 
```
exports.handler = async function http(req) {
  return {
    statusCode: 201,
  }
}
```

I tried to make both `/` and `/login` POST routes to test if a single route (`/`) could not be both GET and POST. However, the problem occurred in all POST routes created with the content shown above.

Using the Insomnia REST client, the timeline feature shows that the following is part of the request:
```
> POST /login HTTP/1.1
> Host: localhost:3333
> User-Agent: insomnia/7.0.1
> Accept: */*
> Content-Length: 0
```

The error message related to the server error is:
```
TypeError: Cannot read property 'includes' of undefined
    at type (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/@architect/sandbox/src/http/index.js:35:44)
    at urlencodedParser (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/body-parser/lib/types/urlencoded.js:98:10)
    at Layer.handle [as handle_request] (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/lib/layer.js:93:5)
    at trim_prefix (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/index.js:335:13)
    at /Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/index.js:299:7
    at Function.process_params (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/index.js:354:12)
    at next (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/index.js:290:10)
    at jsonParser (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/body-parser/lib/types/json.js:119:7)
    at Layer.handle [as handle_request] (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/lib/layer.js:93:5)
    at trim_prefix (/Users/{user}/.nvm/versions/node/v10.15.1/lib/node_modules/@architect/architect/node_modules/router/index.js:335:13)

```

**Steps to reproduce**
1. `arc init` a new project
2. Add a `POST /` route to the `.arc` file under `@http` 
3. Add the minimal code above to the newly generated `post-index/index.js` file
4. Execute a POST request to http://localhost:3333/login

**Expected behavior**
Expected behaviour would be that the `POST /` or `POST /login` requests would not result in an internal server error, but instead return statusCode 201 as intended.

**Desktop**
Please complete the following information (if appropriate):
 - OS: MacOS Mojave 10.14.5
 - Tools: Insomnia and Postman

[edit: removed a word]

**Comment by ryanblock** [comment]
Thanks @andreasbhansen! Good news: fix for this was merged yesterday (see https://github.com/architect/sandbox/issues/102), it'll be in the next release, either tomorrow or Tuesday. In the mean time, the workaround is to ensure your requests include `content-type` header (which they should, per spec).

**Comment by ryanblock** [comment]
PS I <3 Insomnia!

**Comment by andreasbhansen** [comment]
> Thanks @andreasbhansen! Good news: fix for this was merged yesterday (see #102), it'll be in the next release, either tomorrow or Tuesday. In the mean time, the workaround is to ensure your requests include `content-type` header (which they should, per spec).

Perfect, thanks for the fix and workaround!
